pandas/core/common.py

```diff
@@ -479,6 +481,43 @@ def _dict_compat(d):
                 for key, value in iteritems(d))
 
 
+def _standardize_mapping(into):
```

Comment:
doesn't need to be leading underscore, this is a private module. do you have a more informative name? (longer is ok).